src/dal/connector.js

```diff
@@ -2,25 +2,25 @@ import { constants as httpConstants } from 'node:http2'
 import { createLogger } from '../utils/logger.js'
 import { config } from '../config/index.js'
 import { formatDalResponse, mapDalErrors } from './dal-response.js'
+import { getTokenService } from '../services/DAL/token/get-token-service.js'
+import { getTokenCache } from '../server.js'
 
 const logger = createLogger()
 
-export const dalConnector = async (query, variables, email) => {
-  if (!email) {
-    return formatDalResponse({
-      statusCode: httpConstants.HTTP_STATUS_BAD_REQUEST,
-      errors: [{
-        message: 'DAL connection cannot be made if email header is missing'
-      }]
-    })
-  }
+export const dalConnector = async (query, variables) => {
```

Comment:
`dalConenctor` no longer needs the cache passed in as a param as it is imported directly now.